GasolinerasVIP.API/Controllers/UsersController.cs

```diff
@@ -14,7 +18,11 @@ public class UsersController : ControllerBase
         private readonly UserManager<IdentityUser> userManager;
```

Comment:
I think this should be encapsulated in a class.

Reply:
They all ready are un UsersController class.